core/controller/src/main/scala/whisk/core/controller/Activations.scala

```diff
@@ -205,12 +206,37 @@ trait WhiskActivationsApi
     }
 
     /** Custom deserializer for query parameters "since" and "upto" into a valid Instant. */
-    private implicit val stringToInstantDeserializer = new FromStringDeserializer[Instant] {
+    private implicit val stringToInstantDeserializer = new FromStringUnmarshaller[Instant] {
```

Comment:
Are these needed anymore? In general: Try to scrub all the spray dependencies and then remove everything that's related to that.